app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/event_screen/EventScreenEvent.kt

```diff
@@ -0,0 +1,107 @@
+package com.realityexpander.tasky.agenda_feature.presentation.event_screen
+
+import com.realityexpander.tasky.agenda_feature.domain.Attendee
+import com.realityexpander.tasky.agenda_feature.domain.Photo
+import com.realityexpander.tasky.core.presentation.common.util.UiText
+import java.time.ZonedDateTime
+
+sealed interface EventScreenEvent {
```

Comment:
Overall, I find this a bit confusing to be honest. For example, it's not clear to me why there are so many nested sealed interfaces and if these are events sent to the VM or from the VM (for example what is the event `Attendees` used for?)

Reply:
I didnt complete this fully, I hope it will make more sense once I add the rest of the editors. I am attempting to make the calls from the Composable Screen easier to work with and possibly reusable for other screens.